manifests/cf-manifest/spec/manifest/release_versions_spec.rb

```diff
@@ -20,4 +20,27 @@
         "expected release #{release['name']} version #{release['version']} to have matching version in URL: #{release['url']}"
     end
   end
+
+  specify "manifest versions are newer than the ones in cf_deployment" do
```

Comment:
Nitpick: this is actually asserting that they're `>=`, whereas the wording here suggests it's asserting that they're `>`.